api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -20,5 +20,18 @@
     <include file="liquibase-update-to-2.1.xml"/>
     <include file="liquibase-update-to-2.2.xml"/>
     <include file="liquibase-update-to-2.3.xml"/>
-
+<changeSet id="Trunk-5411:Add dose limit unit field to Drug" author="gitacliff">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="drug" columnName="dose_limit_unit"/>				
+			</not>
+		</preConditions>
+		<comment>Adding column dose limit unit in table drug</comment>
```

Comment:
Shouldn't you also add a foreign key constraint?

Reply:
yes @dkayiwa  am giong to add it